ui/src/components/ui/device/device-top-bar/device-top-bar.tsx

```diff
@@ -26,6 +33,21 @@ export const DeviceTopBar = observer(() => {
   const deviceTitle = !device?.ios ? `${device?.manufacturer || ''} ${device?.marketName || ''}` : device?.model || ''
   const currentRotation = `${t('Current rotation:')} ${deviceScreenStore?.getScreenRotation}°`
 
+  useEffect(() => {
+    const onPopState = () => {
```

Comment:
Shouldn't there be an `ev.preventDefault()`?

Reply:
I checked, calling `ev.preventDefault()` would not have any effect. This is because the popstate event is a notification that the browser's history state has already changed, and there is no default action to prevent.